### PR TITLE
PAQ: Use of icons in Streaming Analytics doc

### PR DESCRIPTION
**Pull request by kailol-sag**
**A.** No classes found for:
- document icon (icon-docpane-hidden.png). See https://cumulocity.com/docs/streaming-analytics/analytics-builder/#viewing-the-documentation-for-a-block 
- icons on wires: square resize icon (icon-square-resize.png) + diamond-shaped move icon (icon-diamond-move.png). See https://cumulocity.com/docs/streaming-analytics/analytics-builder/#changing-a-wire

Status update May 10: Sent out a mail to @skom-sag , telling him that these are not web SDK icons and they should therefore be changed to "codex icons" on the dev side.

**B.** Did not change anything in the sections where https://github.com/SoftwareAG/c8y-docs/pull/1938/ is still to be merged. That PR is about the renaming of  "remove" to "delete". The following screenshots have been changed to class definitions on the above PR, but make sure to delete these PNGs from the repo later:
- icon-delete-row.png
- icon-remove-template-parameter.png

Status update May 10: #1938 has been merged and the resulting conflicts have been fixed.

**C.** The changes for the Range Lookup block (move and remove icons) will be done on a later PR. See also https://cumulocity.com/docs/streaming-analytics/analytics-builder/#to-edit-the-parameters-of-a-block . An iTrac for the UI changes will be created for this. icon-move-row.png is still mentioned in 2 places

Status update May 10: Still under development, to be fixed on a later doc PR.

**D.** Found a solution for the expand/collapse icons, but the color is too dark. Is it possible to have the same color as in the screenshot in https://cumulocity.com/docs/streaming-analytics/analytics-builder/#collapsing-and-expanding-a-group ?
- icon-group-collapse.png => dlt-c8y-icon-fullscreen-exit
- icon-group-expand.png => dlt-c8y-icon-fullscreen

Status update May 10: Using "text-primary-light" now, but Hugo currently shows a black icon instead of a lighter teal one. Carlos said that he'll add this class so that the correct color will be shown.

**E.** Need to check everything again later, after all PRs have been merged.

**F.** The doc still uses /icons/user-icon.png.
But why is this an "A"? Because of "apamabld" or "admin" which is often used for our test tenants? Should we reconsider use of that icon? See https://cumulocity.com/docs/streaming-analytics/troubleshooting/

Status update May 10: Removed all references to "images/icons/user-icon.png" from the Streaming Analytics doc.

### CODE REVIEW

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md [hunk 1]
```diff
@@ -663,8 +663,8 @@ The blocks, wires and groups on the canvas always snap to a grid. You can decide
 
 In the model editor, click the toolbar icon for toggling the display of the grid.
```
Comment (MWindel) [inline content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md:664]: @kailol-sag Do we have a toolbar icon here?

File: content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Once an instance is active, you cannot modify the template parameter values or m
 2.  If you have selected simulation mode, click the calendar icon which is now shown, specify the time span that is to be used, and click **Apply**. See also [Simulation parameters](/streaming-analytics/analytics-builder/#simulation-parameters).
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md]: Overlooked it here. Added it :-)

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ It may happen that you and another user are editing the same model at the same t
 
 ##### To save a model {#to-save-a-model}
 
-In the toolbar of the model editor, click the save icon <img src="/images/streaming-analytics/analytics-builder/icon-save.png" alt="Save icon" style="display:inline-block; margin:0">.
+In the toolbar of the model editor, click the save icon <i class="dlt-c8y-icon-save icon-20"></i>.
 
 The save icon is only enabled when changes have been applied to the model and the model has been given a name.
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md]: Ok, added it - even though it is also available in the preceding sentence.

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md (hunk 1 above)
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md:664]: This icon has 2 states and thus looks different in each state. How the icon looks in each state is explained just below this sentence.

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ Any user on the tenant wishing to use the **EPL Apps** page must be a **CEP Mana
 
 ##### Step 1 - Invoke the Streaming Analytics application {#step-1---invoke-the-streaming-analytics-application}
 
-Open the application switcher and click the icon for the **Streaming Analytics** application. Then navigate to the **EPL Apps** page.
+Open the application switcher and click the <i class="c8y-icon c8y-icon-streaming-analytics c8y-icon-duocolor icon-20"></i> icon for the **Streaming Analytics** application. Then navigate to the **EPL Apps** page.
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/epl-apps-bundle/basic-functionality.md:34]: This sounds awkward. 2 times "Streaming Analytics". I'd rather leave it as it is. Or - if you insist - then I'd remove this: 
"icon for the **Streaming Analytics** application". 

Note that your suggestion includes a superfluous "icon" directly after the class definition

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ It is also possible to change the mode from **Inactive** to **Active** (or vice
 Be aware that the EPL editor makes use of a standard web component. It provides many generic developer functions, some of which are not relevant to EPL, including but not limited to Quick Fix and Show Hover.
 {{< /c8y-admon-info >}}
 
-Click **X** in the top menu bar to leave the EPL editor and thus to return to the list of EPL apps.
+Click the <i class="dlt-c8y-icon-clear icon-20"></i> icon in the top menu bar to leave the EPL editor and thus to return to the list of EPL apps.
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/epl-apps-bundle/basic-functionality.md]: Had to check the tooltip in the UI. It now says "close" - they have changed the X tooltips to make them more consistent.

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ It is also possible to change the mode from **Inactive** to **Active** (or vice
 Be aware that the EPL editor makes use of a standard web component. It provides many generic developer functions, some of which are not relevant to EPL, including but not limited to Quick Fix and Show Hover.
 {{< /c8y-admon-info >}}
 
-Click the <i class="dlt-c8y-icon-clear icon-20"></i> icon in the top menu bar to leave the EPL editor and thus to return to the list of EPL apps.
+Click the close <i class="dlt-c8y-icon-clear icon-20"></i> icon in the top menu bar to leave the EPL editor and thus to return to the list of EPL apps.
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:75]:
```suggestion
Click the close icon <i class="dlt-c8y-icon-clear icon-20"></i> in the top menu bar to leave the EPL editor and thus to return to the list of EPL apps.
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**